tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.jsx

```diff
@@ -68,17 +228,33 @@ const SectionSubmissionHistory = ({ section, label, files }) => {
   const pageEnd = Math.min(files.length, pageStart + pageSize)
 
   return (
-    <div className="submission-history-section">
+    <div
+      className="submission-history-section usa-table-container--scrollable"
+      style={{ maxWidth: '100%', margin: '100px' }}
```

Comment:
Nit: Looks like the table is centered, I think if we align it to the left, it will look nicer.
![Screenshot 2023-09-22 at 9 23 42 AM](https://github.com/raft-tech/TANF-app/assets/97037188/fdd122de-8e2f-4fba-8196-557c8c3c5652)

Reply:
im still seeing center-alignment. we should double-check this when moved to gov a11y review. dont want to re-deploy yet @reitermb @ttran-hub @jtimpe